tests/pkg_tinycrypt/main.c

```diff
@@ -0,0 +1,2078 @@
+/* test_aes.c - TinyCrypt AES-128 tests (including NIST tests) */
```

Comment:
same here - as the sources are already extracted to the package source folder in bin/pkg, maybe, with the right makefile (and maybe a small patch), it can be reused from there?

Reply:
Yes, most likely. I think I just have an idea, let me try something...

Reply:
hm, that did not work. Any idea how we could get this done, possibly without patches? Alternatively I tend to simply write a very small test app myselfe, in the end it only needs to to a small number of function calls to the tinycrypt lib, right?